src/rgw/rgw_lc.cc

```diff
@@ -552,7 +549,7 @@ static int remove_expired_obj(
   del_op->params.unmod_since = meta.mtime;
 
   // notification supported only for RADOS driver for now
-  notify = driver->get_notification(dpp, oc.obj.get(), nullptr, event_type,
+  notify = driver->get_notification(dpp, obj.get(), nullptr, event_type,
```

Comment:
how is the change from `oc.obj` to `obj` related to getting the object state?

Reply:
in this commit, I'm just aligning them;  we want to get object state and generate the notification using the same object handle